sdbuild/boot/meta-pynq/recipes-xrt/xrt/xrt_git.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY  = "Xilinx Runtime(XRT) libraries"
+DESCRIPTION = "Xilinx Runtime User Space Libraries and headers"
+
+LICENSE = "GPLv2 & Apache-2.0"
+LIC_FILES_CHKSUM = "file://../LICENSE;md5=fa343562af4b9b922b8d7fe7b0b6d000 \
+                    file://runtime_src/driver/xclng/drm/xocl/LICENSE;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+                    file://runtime_src/driver/xclng/xrt/user_gem/LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+                    file://runtime_src/driver/xclng/tools/xbutil/LICENSE;md5=d273d63619c9aeaf15cdaf76422c4f87"
+
+SRC_URI = "git://github.com/Xilinx/XRT.git;protocol=https;branch=2019.1"
+
+PV = "2.2.0+git${SRCPV}"
+
+# Use latest version
+SRCREV = "${AUTOREV}"
```

Comment:
We probably want to lock these for the release once we have a version that works

Reply:
I will remove this from meta-pynq layer since the recipes are already inside meta-xilinx-bsp layer  in 2019.1 petalinux.